### ISSUE TITLE
Hires texture packs with GlideN64 plugin

### ISSUE THREAD

**Issue by joeauty** [OpenEmu/OpenEmu#4244]
Hello,

I'm able to get the hires texture packs working in vanilla/CLI mupenplus running both Glide and Rice, but this doesn't make for fully ideal gameplay (fullscreen toggle issues, aspect ratio issues, and of course all of the nice OpenEmu features such as console restart, save states, etc. missing). I'd really like to figure out how to get this working in OpenEmu.

I'm assuming that OpenEmu will need to be rebuilt to use any of these other video plugins? GlideN64 does seem to be the most reliable video plugin, but no matter what I do I cannot get my hires packs to load. Is this setting somehow disabled by OpenEmu?

Here is my config, and like I said these same textures in these same paths seem to work fine with both `mupen64plus-video-glide64mk2` and `mupen64plus-video-rice`. Any suggestions?

```
# Use high-resolution texture packs if available.
txHiresEnable = True
# Allow to use alpha channel of high-res texture fully.
txHiresFullAlphaChannel = False
# Use alternative method of paletted textures CRC calculation.
txHresAltCRC = False
# Enable dump of loaded N64 textures.
txDump = False
# Zip textures cache.
txCacheCompression = True
# Force use 16bit texture formats for HD textures.
txForce16bpp = False
# Save texture cache to hard disk.
txSaveCache = True
# Path to folder with hi-res texture packs.
txPath = "/Users/joe/.local/share/mupen64plus/hires_texture"
# Path to folder where plugin saves texture cache files.
txCachePath = "/Users/joe/.cache/mupen64plus/cache"
# Path to folder where plugin saves dumped textures.
txDumpPath = "/Users/joe/.cache/mupen64plus/texture_dump"
```

**Comment by clobber** [comment]
GLideN64 is not the same as glide64mk2. What you want to do is not supported by the plugin.

**Comment by joeauty** [comment]
I'm confused. So, you're saying that even though there are parameters involving hires texture packs, the plugin used by OpenEmu, GlideN64, doesn't support running hires texture packs?

**Comment by joeauty** [comment]
NVM, I'll take it up with the plugin makers.

**Comment by kevinsagis** [comment]
Any news on this in 2026?  Textures work for the plugin GLideN64.  And with Mupen64Plus.  

I would use RetroArch but it's complicated and buggy. I've tried many settings tweaks and it runs too slow, and I have a reasonable computer, 2019 MBP w 8 GB RAM. Only Intel graphics but with an SSD and dual 3.1 Ghz i5 with OS 10.13, it should be able to run an n64 emulator.  I'm a developer but I have tinkered as much as I can with RetroArch and it just won't work.  OpenEmu is basically the wonderful Mac equivalent of Rosalie's emulator, which sadly, is only on Windows.
